### DISCUSSION TITLE
Controll brightness of floodlight cam wired pro?

### DISCUSSION THREAD

**Discussion by tender448** [dgreif/ring#1190] [Ideas]
I just installed one of these and see that I can control the brightness of the lights within the Ring app, but in homekit via homebridge there is just a simple on/off toggle. I know the other smart lights support brightness control, would it be possible to add this for the floodlight cam? I'd love to be able to automate different brightness levels via homekit. Thanks!

**Comment by dgreif** [comment]
It is indeed possible to control brightness levels with homebridge, though I have not set it up for the camera lights in this plugin yet. Do you have some examples of automations in which you would want to control the brightness? My general sense has been that most users would set the brightness to a desired level and just leave it there, which is why this change has not been a high priority

**Comment by tender448** [comment]
I was hoping to find a way to schedule the floodlight to be on a dim setting the the evening and then get brighter when motion is detected, in the same way that the Ring pathlights function. I'm surprised Ring doesn't provide this functionality natively.

**Reply by dgreif** [reply]
I think your best bet would be to request this feature directly from Ring. I agree it's very similar to what they do for the pathlights, and it would make sense for them to add this as an option directly in their app
